### DISCUSSION TITLE
Velocity expression at bondary condition

### DISCUSSION THREAD

**Discussion by AsthLuucas** [ansys/pyfluent#1880] [Q&A]
Can someone help me with the syntax to define a velocity profile at an inlet in pyfluent? I can't seem to get it working

I haven't found anything in the documentation either

Here is the sintax im currently using with constant value
![Capturar](https://github.com/ansys/pyfluent/assets/105306700/03e3952f-6515-4cc7-9a0e-2a81082b5c53)

**Comment by mkundu1** [comment]
This should be very similar to https://github.com/ansys/pyfluent/discussions/1917#discussioncomment-6868745.
